sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -11,6 +11,7 @@
     <TargetFrameworks>$(RequiredTargetFrameworks);net461;netcoreapp2.1;net5.0</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableClientSdkAnalyzers>true</EnableClientSdkAnalyzers>
+    <EnableBannedApiAnalyzers>false</EnableBannedApiAnalyzers>
```

Comment:
Would it be better to `NoWarn` the code rather than disabling it altogether?

Reply:
It might, but this branch was in PR for months and I wanted to get it out.